app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -103,20 +103,25 @@ export const FormAudience = ({
       applicationConfig?.application === experiment.application,
   );
 
-  function parsePercent(value: string | null): number {
-    return parseFloat(value ?? "0");
-  }
+  // function parsePercent(value: string | null): string {
+  //   if (value === "") {
+  //     return value;
+  //   } else {
+  //     let newVal = parseFloat(value).toString;
+  //     return parseFloat(value ?? "").toString();
+  //   }
+  // }
```

Comment:
do you want to keep new function 👀 , it is commented out

Reply:
just fixed it 😂